### DISCUSSION TITLE
How to add a DNS search domain?

### DISCUSSION THREAD

**Discussion by jclagreca** [wg-easy/wg-easy#284] [Q&A]
Previously, I was using OpenVPN, and had an option:  dhcp-option DOMAIN-SEARCH COMPANY.LOCAL.  This way if I ping computer it would resolve computer.company.local.  

In wireguard, I do have a line:  DNS = server.1.ip.addr, server.2.ip.addr, company.local.  

It does resolve computer.company.local, but does not append company.local if I just try to use computer.  

I can't seem to find a way to have a search in wireguard in wg-easy...  Does anyone know how to do this?

**Comment by archont94** [comment]
This container doesn't respond to DNS queries itself - `WG_DEFAULT_DNS` allow you to specify address of external DNS server. So what you can do, is to point it into DNS server which is aware about company.local IP address.
